packages/bahmni-widgets/src/medications/components/actionHandlers.ts

```
@@ -11,15 +11,22 @@ const handleStopAction = (
 
   const encounterUuid = fhirResource.encounter?.reference?.split('/').pop();
 
+  const cancelReasonValueSetUuid = action.metadata?.cancelReasonValueSetUuid;
+  
```

ESLint

[error] Delete `··`

(prettier/prettier)